internal/errors/code.go

```diff
@@ -22,6 +22,7 @@ const (
 
 	// General function errors are reserved Codes 100-999
 	InvalidParameter Code = 100 // InvalidParameter represents and invalid parameter for an operation.
+	Internal         Code = 101 // Internal represents internal errors
```

Comment:
There were a number of errors in the host path that I felt where not specific errors and could not be addressed by the end user, so I felt we should create some sort of internal error when it truely is an internal error.